src/renderer/components/watch-video-playlist/watch-video-playlist.js

```diff
@@ -242,16 +245,45 @@ export default Vue.extend({
       }
     },
 
+    loadCachedPlaylistInformation: async function (cachedPlaylist) {
+      this.isLoading = true
+      this.setCachedPlaylist(null)
```

Comment:
Would this clear the cache when the next video in the playlist loads?

Reply:
Nope, the playlist view just gets hidden with CSS, while changing to a new video in the same playlist ([src/renderer/views/Watch/Watch.vue#L162-L163](https://github.com/FreeTubeApp/FreeTube/blob/176716483df81c8ea138d0d4d568652fa973f55e/src/renderer/views/Watch/Watch.vue#L162-L163)), it doesn't get unmouted and mounted again with new data, like the description or up next views.

This is a good idea, as the playlist view is one of the most expensive components to process and render inside FreeTube, as each ft-list-video element has quite a lot of checks (the amount of checks has increased a lot with #2849 and #2853) to see if it should actually be shown and then Vue also has to create all of those DOM elements (remember each ft-list-video element also has a dropdown menu inside of it), so doing it 872 times, for the large playlist above, is extremely processing intensive (you can notice it if you try to pause or start the video just as vue is inserting all those DOM elements, it will lag for a moment before resuming/pausing the video).

Most video lists in FreeTube have pagination, the playlist view on the watch page on the other hand has to do all of that work in one go.